make_relocatable_python_framework.py

```diff
@@ -83,6 +95,8 @@ def main():
         short_version = ".".join(options.python_version.split(".")[0:2])
         install_extras(
             framework_path,
+            install_wheel=options.install_wheel,
+            upgrade_pip=options.upgrade_pip,
```

Comment:
and then of course change this to reflect the changes in the install_extras function signature.